util.js

```diff
@@ -90,14 +93,11 @@ function spawn (main) {
   return Promise.all(processes.map(p => watch(p)))
 }
 
-const updates = new Map()
+const queryFnMap = new Map()
 
-function getUpdateQuery (count, pg, formats = [BinaryInt]) {
-  const query = updates.get(count)
-  if (query) return query
-  const promise = pg.compile(generateBulkUpdate('world', 'randomnumber', 'id', count, formats))
-  updates.set(count, promise)
-  return promise
+const query = async (count, pg, ...queryArgs) => {
```

Comment:
reduced unneeded assignments as the set get calls get replaced by v8 internal with the correct pointers anyway.